crates/repository/src/lib.rs

```diff
@@ -139,26 +140,18 @@ impl Repository {
         target: &str,
         passthrough_arguments: &[String],
     ) -> Result<()> {
-        let mut shell_command = String::new();
-
-        if target == "nao" {
-            shell_command += &format!(
-                ". {} && ",
-                self.root
-                    .join(format!(
-                        "naosdk/{SDK_VERSION}/environment-setup-corei7-64-aldebaran-linux"
-                    ))
-                    .display()
-            );
-        }
-
         let cargo_command = format!("cargo {action} ")
             + format!("--profile {profile} ").as_str()
             + if workspace {
                 "--workspace --all-features --all-targets ".to_string()
             } else {
                 let manifest = format!("crates/hulk_{target}/Cargo.toml");
-                format!("--manifest-path={} ", self.root.join(manifest).display())
+                let root = if target == "nao" && USE_DOCKER {
```

Comment:
```suggestion
                let root = if use_docker {
```

and introduce the `use_docker` variable accordingly

Reply:
Thanks, done.